frontend/src/Utils/Api/APIService.tsx

```diff
@@ -1,5 +1,7 @@
+import { LoginProp, RegisterProp, CreateMessageProp, CreateRoomProp } from 'Utils/types';
+
 export default class APIService {
-  static LoginUser({ email, password }) {
+  static LoginUser({ email, password }: LoginProp) {
```

Comment:
`LoginProps` => `LoginUserConfiguration`

`LoginProps` is not an component, so it's arguments are not props